test/integration/targets/mysql_query/tasks/mysql_query_initial.yml

```diff
@@ -0,0 +1,332 @@
+# Test code for mysql_query module
+# Copyright: (c) 2020, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+###################
+# Prepare for tests
+#
+
+# Prepare SQL script:
+- name: Remove SQL script if exists
+  become: yes
+  file:
+    path: '{{ test_script_path }}'
+    state: absent
+  ignore_errors: yes
+
+- name: Create test sql script
+  become: yes
+  file:
+    path: '{{ test_script_path }}'
+    state: touch
+    mode: 0644
+
+- name: Prepare SQL script
+  become: yes
+  shell: 'echo "{{ item }}" >> {{ test_script_path }}'
+  ignore_errors: yes
+  loop:
+  - 'INSERT INTO {{ test_table1 }} (id) VALUES (1), (2), (3);'
```

Comment:
One of the most important things we need to ensure, is that the module supports executing multiple queries at onces, which isn't necessarily the case.

Reply:
i'll remove the option at all, thanks. We have mysql_db import

Reply:
My previous comment also applies to `query` including `;` / multiple queries at once 😉

Reply:
😀 @bmalynovytch , it should work, I believe, in the form of string “query1; query2;”.
If you mean passing a list as a query value, it seems that it’s a problem to use pos_args and named_args with it.
I’ll check the string form with ; tomorrow